### DISCUSSION TITLE
Block model movement in AR after placement

### DISCUSSION THREAD

**Discussion by danilokrulj** [google/model-viewer#3452] [Q&A]
Guys is it possible to block model movement while in AR? The idea is to let the user place the model where he wants and than confirm the models position via button click, after that I would like to block further model movement (by the user) and display some other buttons/actions (after he confirms he is pleased with the models position and confirms it via button click). I tried using a div to cover the whole screen but that only blocks interaction with other UI elements, the model still reacts to touch input.

**Comment by samaneh-kazemi** [comment]
Hi @danilokrulj, this is not possible. For instance if you're using Android, Scene-Viewer would do that for us and we don't control that
